### ISSUE TITLE
Exclude dynamically changing data from text

### ISSUE THREAD

**Issue by hotgazpacho** [approvals/ApprovalTests.Ruby#54]
There currently exists a way to exclude dynamically changing data (e.g. dates) from JSON. It would be nice to extend this to text documents (and maybe HTML) as well.

**Comment by mariocsee** [comment]
Has there been any updates to this issue?

**Comment by kytrinyx** [comment]
No, there hasn't been any work done on it. I don't actively develop the gem anymore, though I do review and discuss proposed patches.

**Comment by geeksam** [comment]
Oh, hi!  I've been working on a comparison tool specifically designed for JSON data that supports ignoring specific parts of the document.  It's still wet around the ears and definitely not worth adding as a dependency to this gem... but maybe an adapter can be written?  Or, of course, feel free to swipe any ideas and/or code that's useful.  :)

https://github.com/realgeeks/check_please

**Comment by geeksam** [comment]
Oh, see also https://github.com/mcmire/super_diff/
